general/1_terminal/Yevhenii-Bykov/Task_1_2.md

```diff
@@ -1,14 +1,14 @@
-# Task 1,2
-### Task_1
-
-To output the name of files that have `zoola` in their names i used:
-
-```sh
-find ./ -name "*zoola*"
-```
-And i get files from the current folder that have `zoola` in their name.
-
-## Task2
-> The nested directory is created, but the working directory didn't change. Why? 
-
+# Task 1,2
```

Comment:
please rollback changes that are not related to your project

Reply:
How do I do that? Should I just remove the files? If yes, which exactly? 
Sorry for such dumb questions.

Reply:
since it's just line separators, let's keep it as is

but please remove all files from 6,7,8 homeworks - specific branches have more actual changes